test/Microsoft.AspNetCore.Mvc.FunctionalTests/compiler/resources/RazorPagesWebSite.SimpleForms.html

```diff
@@ -0,0 +1,5 @@
+<form></form>
+<form method="get"></form>
+<form method="post"><input name="__RequestVerificationToken" type="hidden" value="{0}" /></form>
+<form action="/Foo/Bar/Baz.html" method="get"></form>
+<form action="/Foo/Bar/Baz.html" method="post"></form>
```

Comment:
@dougbu
> Tag helper should have added an antiforgery token, changing the behaviour when user includes `action` attribute. Need to make a change around [line 168 in the tag helper](https://github.com/aspnet/Mvc/blob/b4b315146594f0ba2b28fbd47309290ea06a5b7a/src/Microsoft.AspNetCore.Mvc.TagHelpers/FormTagHelper.cs#L168) to match the updated [line 248](https://github.com/aspnet/Mvc/blob/b4b315146594f0ba2b28fbd47309290ea06a5b7a/src/Microsoft.AspNetCore.Mvc.TagHelpers/FormTagHelper.cs#L248).

@NTaylorMullen 
> Talked with Damian about this. Since the user specified `action`, they're opting out of our TagHelper ways. Therefore, don't generate a token.

Let's chat quickly tomorrow. I agree the user has opted out of the tag helper generating an `action` in this case. But, I'm not sure why `<form action="" method="post">` is any more an opt out of antiforgery than `<form method="post">` is -- both submit the same data to the same URL.